dlc_developer_config.toml

```diff
@@ -20,7 +20,7 @@ huggingface_trcomp_mode = false
 # Please only set it to True if you are preparing a TRCOMP related PR
 # Do remember to revert it back to False before merging any PR (including TRCOMP dedicated PR)
 # This mode is used to build PT1.12 and above DLC
-trcomp_mode = false
+trcomp_mode = true
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
```

Comment:
Set the `sagemaker_remote_tests` flag to `standard` to enable SM test

Reply:
Done